cli/common.cpp

```diff
@@ -290,6 +290,8 @@ cli::get_result_types(const utils::cmdline::parsed_cmdline& cmdline)
 ///
 /// \param args The command-line arguments representing test filters.
 ///
+/// \return a set of test filters.
```

Comment:
Capital letter at the beginning of a sentence. Same thing throughout the change.